locales/es.yaml

```diff
@@ -44,47 +44,47 @@ term:
   internalLevel: Nivel interno
   minInternalLevel: Nivel mínimo interno
   maxInternalLevel: Nivel máximo interno
-  noteDesigner: Notas del Diseñador
-  totalNotes: TOTAL
+  noteDesigner: Diseñador del mapa
+  totalNotes: Total
   region: Región
 
 # Descriptions & Tooltips
 description:
   pending: Pendiente
   loading: Cargando datos…
   newSong: ¡Nueva canción!
-  unlockNeeded: Hay que desbloquear esta canción
+  unlockNeeded: Hay que desbloquear esta canción.
```

Comment:
This entry is used as the tooltip of locked sheets in the Sheet Dialog.
The English translation is intended to be short and without a period, just like the below `unavailableInRegion` one.